admin_guide/allocating_node_resources.adoc

```diff
@@ -120,57 +120,61 @@ to measure performance metrics of your deployment at various cluster states.
 An allocated amount of a resource is computed based on the following formula:
 
 ----
-[Allocatable] = [Node Capacity] - [kube-reserved] - [system-reserved] - [Hard-Eviction-Thresholds]
+[Allocatable] = [Node Capacity] - [system-reserved] - [Hard-Eviction-Thresholds]
 ----
 
 [NOTE]
 ====
-The withholding of `Hard-Eviction-Thresholds` from allocatable is a change in behavior to improve
-system reliability now that allocatable is enforced for end-user pods at the node level.
-The `*experimental-allocatable-ignore-eviction*` setting is available to preserve legacy behavior,
-but it will be deprecated in a future release.
+The withholding of `Hard-Eviction-Thresholds` from allocatable improves system reliability because the value for allocatable is enforced for pods at the node level. The `experimental-allocatable-ignore-eviction` setting is available to preserve legacy behavior, but it will be deprecated in a future release.
 ====
 
-
-If `[Allocatable]` is negative, it is set to *0*.
+If `[Allocatable]` is negative, it is set to `0`.
 
 [[viewing-node-allocatable-resources-and-capacity]]
-== Viewing Node Allocatable Resources and Capacity
+== Viewing Node-Allocatable Resources and Capacity
 
-To see a node's current capacity and allocatable resources, you can run:
+To view the current capacity and allocatable resources for a node, you can run the following command:
```

Comment:
Not huge but I notice a couple of instances, such as here, where we say "you can run" when "run" would suffice.